control-plane-operator/controllers/hostedcontrolplane/cloud/azure/providerconfig.go

```diff
@@ -85,6 +112,7 @@ type AzureConfig struct {
 	VnetResourceGroup            string `json:"vnetResourceGroup"`
 	SubnetName                   string `json:"subnetName"`
 	SecurityGroupName            string `json:"securityGroupName"`
+	SecurityGroupResourceGroup   string `json:"securityGroupResourceGroup"`
```

Comment:
Should we leverage a resource ID for the NSG and then plumb this all the way in?  Then we make no guarantee that the cluster NSG is required to be in the network resource group.

Reply:
Ignore; separate card/item.  https://issues.redhat.com/browse/HOSTEDCP-1557